### DISCUSSION TITLE
Add shipping costs to Avalara lines

### DISCUSSION THREAD

**Discussion by Anubiella** [saleor/saleor#9941] [General]
**Hi everyone,**

_**thanks again for the last fix about the Avalara plugin issue for net/gross prices, it was a huge step!**_

Now our stakeholders are asking for the chance to send shipping costs line along with
other products lines to Avalara.

Avalara links just to get the idea of what we're trying to achieve:
https://developer.avalara.com/avatax/dev-guide/shipping-and-handling/
https://developer.avalara.com/avatax/dev-guide/shipping-and-handling/taxability-of-shipping-charges/

Is it already possible?
If not, are you planning to implement this feature?

Thanks a lot in advance,
god bless

**Comment by timuric** [comment]
@Anubiella we are planning to rewrite Avalara to a Saleor app (written in typescript and optimized for deployment in vercel), which would be friendly for forking. We will consider adding this feature.
